### PR TITLE
Update dsintro.rst

### PR DESCRIPTION
**Pull request by yoonghm**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] Added examples to create DataFrame from series and series with `nan`

### CODE REVIEW

**Comment by yoonghm** [comment]
For your review.
There should another method to create DataFrame from numpy series which has no index.

File: doc/source/dsintro.rst [hunk 1]
```diff
@@ -366,13 +366,32 @@ dictionary.
 
 .. _basics.dataframe.from_series:
 
-From a Series
-~~~~~~~~~~~~~
+From a Panda Series
```
Comment (jorisvandenbossche) [inline doc/source/dsintro.rst]: Panda -> pandas

File: doc/source/dsintro.rst [hunk 2]
```diff
@@ -366,13 +366,32 @@ dictionary.
...
+~~~~~~~~~~~~~~~~~~~
 
 The result will be a DataFrame with the same index as the input Series, and
 with one column whose name is the original name of the Series (only if no other
 column name provided).
 
+.. ipython:: python
+
+   df = pd.DataFrame()
```
Comment (jorisvandenbossche) [inline doc/source/dsintro.rst]: We typically don't want to encourage creating an empty dataframe first, and then filling it. 

You can give an example of directly creating a DataFrame from a Series like:

```
s = pd.Series([1, 2, 3])
pd.DataFrame(s)
```

File: doc/source/dsintro.rst [hunk 3]
```diff
@@ -366,13 +366,32 @@ dictionary.
...
+   df
+   x1 = pd.Series([1,2,3,4], index=[1,2,3,4])
+   x1
+   df['A'] = x1
+   df
+   x2 = pd.Series([5,5], index=[1, 3])
+   x2
+   df['B'] = x2
```
Comment (jorisvandenbossche) [inline doc/source/dsintro.rst]: This is more an example of how to add a column to existing dataframe, and how the values get aligned first before getting added. 
So I think it is not fully fitting for here (this section is only about creating a DataFrame)

Comment (yoonghm) [reply doc/source/dsintro.rst]: Okay, I will update that.

Comment (yoonghm) [reply doc/source/dsintro.rst]: Is there a place I could document
- Adding/replacing a column from a numpy series with more than 1 columns
- Adding/replacing multiple columns from a numpy series with multiple columns.

Please reject the pull request.

Comment (jorisvandenbossche) [reply doc/source/dsintro.rst]: There is already a section in this file about adding columns: http://pandas.pydata.org/pandas-docs/stable/dsintro.html#column-selection-addition-deletion

**Comment by yoonghm** [comment]
Hi,

  x2 is a ndarrays of 5 x 5. When I assign it to df['B'], only the first
column of x2 is included and there is warning whatsoever. Is this feature
documented?

File: doc/source/dsintro.rst [hunk 4]
```diff
@@ -366,13 +366,56 @@ dictionary.
...
+
+Numpy series is list or list of lists with no index.
+
+.. ipython:: python
+
+    x1 = np.random.randint(0, 10, (5))
+    x1
+    x2 = np.random.randint(-1.0, 10, (5))
+    x2
+    df1 = pd.DataFrame(x1, columns=['A'])
```
Comment (jreback) [inline doc/source/dsintro.rst]: yeah i would rather have these examples in the doc-string of the Series/DataFrame constructor than here

**Review by jreback — CHANGES_REQUESTED** [review]

**Pull request closed without merging**